app-text/ghostscript_gpl/ghostscript_gpl-9.24.recipe

```diff
@@ -7,53 +7,54 @@ the conversion between PostScript and PDF files."
 HOMEPAGE="https://ghostscript.com/"
 COPYRIGHT="2001-2018 Artifex Software, Inc."
 LICENSE="AGPL-3.0"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs924/ghostscript-$portVersion.tar.gz"
 CHECKSUM_SHA256="e613b2376f01882e82e0cf08f6caa3c5719793c4e31dd7d4a8e87d61d371005f"
 SOURCE_DIR="ghostscript-$portVersion"
 
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+fi
 
 # On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
-commandSuffix=$secondaryArchSuffix
 commandBinDir=$binDir
 if [ "$targetArchitecture" = x86_gcc2 ]; then
-	commandSuffix=
 	commandBinDir=$prefix/bin
 fi
 
 PROVIDES="
 	ghostscript_gpl$secondaryArchSuffix = $portVersion
-	cmd:dvipdf$commandSuffix
-	cmd:eps2eps$commandSuffix
-	cmd:font2c$commandSuffix
-	cmd:gs$commandSuffix
-	cmd:gsbj$commandSuffix
-	cmd:gsdj$commandSuffix
-	cmd:gsdj500$commandSuffix
-	cmd:gslj$commandSuffix
-	cmd:gslp$commandSuffix
-	cmd:gsnd$commandSuffix
-	cmd:lprsetup.sh$commandSuffix
-	cmd:pfbtopfa$commandSuffix
-	cmd:pdf2dsc$commandSuffix
-	cmd:pdf2ps$commandSuffix
-	cmd:pf2afm$commandSuffix
-	cmd:pphs$commandSuffix
-	cmd:printafm$commandSuffix
-	cmd:ps2ascii$commandSuffix
-	cmd:ps2epsi$commandSuffix
-	cmd:ps2pdf$commandSuffix
-	cmd:ps2pdfwr$commandSuffix
-	cmd:ps2pdf12$commandSuffix
-	cmd:ps2pdf13$commandSuffix
-	cmd:ps2pdf14$commandSuffix
-	cmd:ps2ps$commandSuffix
-	cmd:ps2ps2$commandSuffix
-	cmd:unix_lpr.sh$commandSuffix
-	cmd:wftopfa$commandSuffix
+	cmd:dvipdf
+	cmd:eps2eps
+	cmd:font2c
+	cmd:gs
+	cmd:gsbj
+	cmd:gsdj
+	cmd:gsdj500
+	cmd:gslj
+	cmd:gslp
+	cmd:gsnd
+	cmd:lprsetup.sh
+	cmd:pfbtopfa
+	cmd:pdf2dsc
+	cmd:pdf2ps
+	cmd:pf2afm
+	cmd:pphs
+	cmd:printafm
+	cmd:ps2ascii
+	cmd:ps2epsi
+	cmd:ps2pdf
+	cmd:ps2pdfwr
+	cmd:ps2pdf12
+	cmd:ps2pdf13
+	cmd:ps2pdf14
+	cmd:ps2ps
+	cmd:ps2ps2
+	cmd:unix_lpr.sh
+	cmd:wftopfa
 	"
+
```

Comment:
Looks OK, with a tiny change: Removing line 57.
```diff
 	cmd:wftopfa
 	"
-
 REQUIRES="
 	haiku$secondaryArchSuffix
```
(We prefer not to have an empty line before `REQUIRES`.)
[HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Let me check that `9.24` builds fine on x86 second arch, and then I'll merge. (It should build.)